### ISSUE TITLE
PatchTST tutorial missing data

### ISSUE THREAD

**Issue by emanuelshalev** [timeseriesAI/tsai#871]
The PatchTST tutorial is trying to download data from a location no longer available.

In the "Load and Prepare Data" section of the tutorial it says: You can download all data from here: https://cloud.tsinghua.edu.cn/d/e1ccfff39ad541908bae/

The URI is no longer available. And the code to download data fails.

**Comment by profPlum** [comment]
Yes please fix this is the only tutorial for time-series forecasting.

**Comment by DomagojKorais** [comment]
you can find the dataset here: https://github.com/OrigamiSL/GBT?tab=readme-ov-file

**Comment by zhangL2auto** [comment]
> you can find the dataset here: https://github.com/OrigamiSL/GBT?tab=readme-ov-file

I can NOT find the data in this link
